+ source ../common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Wed Nov 22 05:41:19 2023 - INFO  - Start to prepare the test environment.
+ DNF_INSTALL cri-o
+ pkgs=cri-o
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs cri-o --node 1 --tempfile ''
+ tmpfile2='Last metadata expiration check: 1 day, 3:52:29 ago on Tue 21 Nov 2023 01:49:15 AM CST.
No match for argument: cri-o
Error: Unable to find a match: cri-o'
+ '[' -z '' ']'
+ tmpfile='Last metadata expiration check: 1 day, 3:52:29 ago on Tue 21 Nov 2023 01:49:15 AM CST.
No match for argument: cri-o
Error: Unable to find a match: cri-o'
+ systemctl start crio-wipe.service
Failed to start crio-wipe.service: Unit crio-wipe.service not found.
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Wed Nov 22 05:41:46 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Wed Nov 22 05:41:46 2023 - INFO  - Start to run test.
+ test_oneshot crio-wipe.service 'inactive (dead)'
+ service=crio-wipe.service
+ status='inactive (dead)'
+ systemctl status crio-wipe.service
+ grep Active
+ grep -v 'inactive (dead)'
Unit crio-wipe.service could not be found.
+ CHECK_RESULT 1 0 1 'There is an error for the status of crio-wipe.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error for the status of crio-wipe.service'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ test_enabled crio-wipe.service
+ service=crio-wipe.service
++ systemctl is-enabled crio-wipe.service
Failed to get unit file state for crio-wipe.service: No such file or directory
+ state=
+ '[' '' == enabled ']'
+ '[' '' == disabled ']'
+ '[' '' == masked ']'
+ '[' '' == static ']'
+ LOG_INFO 'Unit is indirect, ignoring.'
+ message='Unit is indirect, ignoring.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Unit is indirect, ignoring.'
Wed Nov 22 05:41:47 2023 - INFO  - Unit is indirect, ignoring.
+ journalctl -u crio-wipe.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 0 1 'There is an error message for the log of crio-wipe.service'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of crio-wipe.service'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Wed Nov 22 05:41:47 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z ''
+ test 0 -eq 0
+ LOG_INFO 'succeed to execute the case.'
+ message='succeed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'succeed to execute the case.'
Wed Nov 22 05:41:48 2023 - INFO  - succeed to execute the case.
+ exec_result=
+ exit 0
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Wed Nov 22 05:41:48 2023 - INFO  - Start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z Last metadata expiration check: 1 day, 3:52:29 ago on Tue 21 Nov 2023 01:49:15 AM CST.
No match for argument: cri-o
Error: Unable to find a match: cri-o ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile 'Last metadata expiration check: 1 day, 3:52:29 ago on Tue 21 Nov 2023 01:49:15 AM CST.
No match for argument: cri-o
Error: Unable to find a match: cri-o'
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/rpm_manage.py", line 192, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen/libs/locallibs/rpm_manage.py", line 163, in rpm_remove
    with open(tmpfile, "r") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'Last metadata expiration check: 1 day, 3:52:29 ago on Tue 21 Nov 2023 01:49:15 AM CST.\nNo match for argument: cri-o\nError: Unable to find a match: cri-o'
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Wed Nov 22 05:41:51 2023 - INFO  - End to restore the test environment.
